src/test/java/org/jboss/xavier/integrations/EndToEndTest.java

```diff
@@ -0,0 +1,524 @@
+                        .withEnv("INGRESS_VALIDTOPICS","xavier,testareno,advisortestareno,advisor")
+                        .withEnv("OPENSHIFT_BUILD_COMMIT","woopwoop")
+                        .withEnv("INGRESS_MINIODEV","true")
+                        .withEnv("INGRESS_MINIOACCESSKEY","BQA2GEXO711FVBVXDWKM")
+                        .withEnv("INGRESS_MINIOSECRETKEY","uvgz3LCwWM3e400cDkQIH/y1Y4xgU4iV91CwFSPC")
+                        .withEnv("INGRESS_MINIOENDPOINT", "minio:9000")
+                        .withEnv("INGRESS_KAFKABROKERS", "kafka:9092");
+                ingress.start();
+
+                importProjectIntoKIE();
+
+                EnvironmentTestUtils.addEnvironment("environment", configurableApplicationContext.getEnvironment(),
+                        "amq.server=" + activemq.getContainerIpAddress(),
+                        "amq.port=" + activemq.getMappedPort(61616),
+                        "minio.host=" + getContainerHost(minio, 9000),
+                        "insights.upload.host=" + getContainerHost(ingress),
+                        "insights.properties=yearOverYearGrowthRatePercentage,percentageOfHypervisorsMigratedOnYear1,percentageOfHypervisorsMigratedOnYear2,percentageOfHypervisorsMigratedOnYear3,reportName,reportDescription",
+                        "camel.component.servlet.mapping.context-path=/api/xavier/*",
+                        "insights.kafka.host=" + kafka.getBootstrapServers(),
+                        "postgresql.service.name=" + postgreSQL.getContainerIpAddress(),
+                        "postgresql.service.port=" + postgreSQL.getFirstMappedPort(),
+                        "spring.datasource.username=" + postgreSQL.getUsername(),
+                        "spring.datasource.password=" + postgreSQL.getPassword(),
+                        "S3_HOST=" + localstack.getEndpointConfiguration(S3).getServiceEndpoint(),
+                        "S3_REGION="+ localstack.getEndpointConfiguration(S3).getSigningRegion(),
+                        "kieserver.devel-service=" + getHostForKie() + "/kie-server",
+                        "spring.datasource.url = jdbc:postgresql://" + getContainerHost(postgreSQL) + "/sampledb" ,
+                        "spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQL9Dialect");
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+
+        @NotNull
+        private String getContainerHost(GenericContainer container, Integer port) {
+            return container.getContainerIpAddress() + ":" + container.getMappedPort(port);
+        }
+
+        @NotNull
+        private static String getContainerHost(GenericContainer container) {
+            return container.getContainerIpAddress() + ":" + container.getFirstMappedPort();
+        }
+    }
+
+    @Inject
+    CamelContext camelContext;
+
+    @Inject
+    JmsTemplate jmsTemplate;
```

Comment:
We can remove this since it is not used anywhere

Reply:
+1